plugin/CactbotEventSource/CactbotEventSource.cs

```diff
@@ -624,6 +624,17 @@ struct OverlayPreset : IOverlayPreset {
       });
     }
 
+    private void RegisterExternalPreset(string name, string url, int width, int height, bool locked)
+    {
+        Registry.RegisterOverlayPreset(new OverlayPreset
+        {
+            Name = $"Cactbot External {name}",
+            Url = url,
+            Size = new int[] { width, height },
+            Locked = locked,
```

Comment:
I think this should probably just be `Locked = false` (with no function param) like the other ones until there's an example that needs it.

Reply:
That's fair, I'm kinda used to leaving things open as an option but I get it.